stages/14-director-gui-check/gui-dmaas-device-minio

```diff
@@ -15,8 +15,8 @@ cp  $path/.awscluster2/config ~/.kube/config
 # ####################################
 # ##  Sequencing and Running test   ##
 # ####################################
-# bash utils/pooling jobname:tcid-dir-dmaas-schd-cstor-minio-bucket
-# bash utils/e2e-cr jobname:tcid-dir-dmaas-schd-local-dev-minio-bucket jobphase:Running
+bash utils/pooling jobname:tcid-dir-dmaas-schd-cstor-minio-bucket
+bash utils/e2e-cr jobname:tcid-dir-dmaas-schd-local-dev-minio-bucket jobphase:Running
```

Comment:
We are doing almost the same thing everywhere. However, its not understood clearly.
Probably as a first step these should have enough code comments.
Second step is to think on ways to make these declarative.

Reply:
Yes , we are doing same thing because every job needs to run in sequence.